code/game/machinery/vending.dm

```diff
@@ -814,31 +814,55 @@
 	icon_state = "cart"
 	icon_deny = "cart-deny"
 	req_access = list(access_hop)
-	products = list(/obj/item/weapon/cartridge/medical = 10,/obj/item/weapon/cartridge/engineering = 10,/obj/item/weapon/cartridge/security = 10,
-					/obj/item/weapon/cartridge/janitor = 10,/obj/item/weapon/cartridge/signal/science = 10,/obj/item/device/pda/heads = 10,
-					/obj/item/weapon/cartridge/captain = 3,/obj/item/weapon/cartridge/quartermaster = 10)
+	products = list(
+		/obj/item/weapon/cartridge/captain = 3,
+		/obj/item/weapon/cartridge/hop = 3,
+		/obj/item/weapon/cartridge/cmo = 3,
+		/obj/item/weapon/cartridge/medical = 5,
+		/obj/item/weapon/cartridge/chemistry = 5,
+		/obj/item/weapon/cartridge/ce = 3,
+		/obj/item/weapon/cartridge/engineering = 5,
+		/obj/item/weapon/cartridge/atmos = 5,
+		/obj/item/weapon/cartridge/mechanic = 5,
+		/obj/item/weapon/cartridge/rd = 3,
+		/obj/item/weapon/cartridge/signal/toxins = 5,
+		/obj/item/weapon/cartridge/robotics = 5,
+		/obj/item/weapon/cartridge/hos = 3,
+		/obj/item/weapon/cartridge/security = 5,
+		/obj/item/weapon/cartridge/detective = 5,
+		/obj/item/weapon/cartridge/lawyer = 5,
+		/obj/item/weapon/cartridge/quartermaster = 5,
+		/obj/item/weapon/cartridge/chef = 5,
+		/obj/item/weapon/cartridge/janitor = 5)
+		)

```

Comment:
Have an extra 'end parenthesis' there.

Reply:
Yeah, I know. Waiting to fix until input on actual additions so can make one commit since web editor. Thanks, though.